tests/test_cli.py

```diff
@@ -71,9 +71,12 @@ def _parse_output(output: str, output_format: str) -> Any:
         For table format, a list of dict formatted rows.
         For JSON or YAML format, a list or a dict corresponding to ``output``.
     """
-
-    if output_format == "table":
-        rows = output.split(os.linesep)
+    if output_format == "value":
+        # Currently, _parse_output with output_format="value" is used only for
+        # `study-names` command.
+        return [{"name": values} for values in output.split("\n")]
```

Comment:
Could you use `os.linesep` instead of `\n`? I think that Windows uses `\r\n` as the line separator in the output of CLI command.

Reply:
It seems that just replacing `os.linesep` does not resolve the error since the original implementation used them as in https://github.com/optuna/optuna/pull/5029/commits/e95c13a15e98675dc1545340b994227922876d7d. I'll look for the cause.